kubernetes/samples/scripts/create-oracle-db-service/start-db-service.sh

```diff
@@ -6,28 +6,32 @@
 
 script="${BASH_SOURCE[0]}"
 scriptDir="$( cd "$( dirname "${script}" )" && pwd )"
-source ${scriptDir}/common/utility.sh
+source ${scriptDir}/../common/utility.sh
 
 function usage {
-  echo "usage: ${script} -p <nodeport> -i <image> -s <docker-store> [-h]"
+  echo "usage: ${script} -p <nodeport> -i <image> -s <docker-store> -n <name-space>  [-h]"
   echo "  -i  Oracle DB Image (optional)"
   echo "      (default: container-registry.oracle.com/database/enterprise:12.2.0.1-slim ) "
   echo "  -p DB Service NodePort (optional)"
   echo "      (default: 30011) "
   echo "  -s DB Image PullSecret  (optional)"
   echo "      (default: docker-store) "
+  echo "  -n Kubernate Namespace for Oracle DB Service  (optional)"
```

Comment:
Right... do note above that the namespace is configurable.

Reply:
Modified the description 
echo "  -n Configurable Kubernate Namespace for Oracle DB Service  (optional)"
  echo "      (default: default) "